source/AI.cpp

```diff
@@ -1256,6 +1261,8 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 					target.reset();
 					break;
 				}
+		if(!target)
+			ship.SetFleeing();
```

Comment:
Not finding a target is not sufficient for me to conclude that some ship is fleeing; ships that are fleeing should not be doing any `FindTarget()` calls.
Isn't there a better location, like for example the location where a coward personality actually decides to flee?

Reply:
We are precisely at the location where said personality, or any other reason makes a ship flee, the target is only reset if the ship decides to flee because it has somewhere to flee to, if I'm not mistaken.

Reply:
To be clear, any other ship not having a target but not being in the process of fleeing will in fact not be considered fleeing